features/upgrade/storage/upgrade.feature

```diff
@@ -344,3 +344,120 @@ Feature: Storage upgrade tests
       | ls | /mnt/hostpath |
     Then the output should contain:
       | test-before-upgrade |
+
+  # @author ropatil@redhat.com
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Snapshot operator should work well before and after upgrade of cluster on AWS platform - prepare
+    Given I switch to cluster admin pseudo user
+    Given the master version >= "4.7"
+      
+    #Snapshot operator/controller status checking
+    Given the "csi-snapshot-controller" operator version matches the current cluster version
+
+    Given the status of condition "Degraded" for "csi-snapshot-controller" operator is: False
+    Given the status of condition "Progressing" for "csi-snapshot-controller" operator is: False
+    Given the status of condition "Available" for "csi-snapshot-controller" operator is: True
+    Given the status of condition "Upgradeable" for "csi-snapshot-controller" operator is: True
+
+    When I run the :new_project client command with:
+      | project_name  | snapshot-upgrade-ocp-44578  |
```

Comment:
Style suggestion: make sure there is only one space between `project_name` and `|` in the table.